load_testing/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains scripts for running load tests. The framework we are using is called Locust,
+which is an open source program written in Python. It's very simple to setup and get going,
+which is the main reason why we chose it.
+
+Runs on:
+Python 3
+
+To install requirements run:
+
+'pip install -r requirements.txt'
+
+Save a copy of  'loadTestConstants.EXAMPLE.py' as 'loadTestConstants.py'
```

Comment:
Can you update this to accomodate for .env instead of the .py file :)

Reply:
fixed typos, added .env to read me